README.md

```diff
@@ -35,6 +35,7 @@ pip install -e .     # install python dependencies
 ### Running the app
 
 ```
+export PYTHONPATH="${PYTHONPATH}:/path/to/structure-property-visualizer"  # export pythonpath
```

Comment:
Just a question: Is this really needed also when one runs this _from_ the folder of the repository?
I always thought the current working directory is added to the PYTHONPATH automatically

Reply:
I was also surprised, but when I ran it inside the structure-property-visualizer folder, it complained that it couldn't import the modules:
`2022-09-14 22:16:21,702 Error running application handler <bokeh.application.handlers.directory.DirectoryHandler object at 0x7f8ff3dc2580>: No module named 'import_db'
File "main.py", line 15, in <module>:
from import_db import get_cif_content_from_disk as get_cif_str Traceback (most recent call last):
  File "/home/lsmo/miniconda/envs/test/lib/python3.9/site-packages/bokeh/application/handlers/code_runner.py", line 179, in run
    exec(self._code, module.__dict__)
  File "/home/lsmo/test/structure-property-visualizer/detail/main.py", line 15, in <module>
    from import_db import get_cif_content_from_disk as get_cif_str
ModuleNotFoundError: No module named 'import_db'`

Reply:
exporting the path fixed this for me and Elias

Reply:
I see, thanks!